src/app/transaction/UserOperation.tsx

```diff
@@ -497,6 +510,155 @@ function UserOperation({ transactionData }: any) {
                   </TableRow>
                 </TableContainer>
               </SmartRow>
+              {data?.erc721Transfers?.length > 0 && (
```

Comment:
Can we put them in a separate component?

Reply:
Yes we can put them in separate component. Will do and push the code.